MoneySplitter.Infrastructure/INavigationManager.cs

```diff
@@ -7,9 +7,12 @@ public interface INavigationManager
         void NavigateToShellViewModel();
         void NavigateToRegisterViewModel();
         void InitializeShellNavigationService(object navigationService);
-        void NavigateToMainPage();
+        void NavigateToHome();
         void NavigateToFriends();
         void NavigateToShellViewModel(Type viewModelType);
```

Comment:
use same convension. 

NavigateTo{name}ViewModel 
or NavigateTo{name}

should be single convention